src/plugins/intel_gna/legacy/src/convert_function_to_cnn_network.cpp

```diff
@@ -481,7 +483,7 @@ void CNNLayerCreator::on_adapter(const std::string& name, ::ngraph::ValueAccesso
     }
 }
 
-CNNLayerCreator::CNNLayerCreator(const std::shared_ptr<::ngraph::Node>& node) : node(node) {
+CNNLayerCreator::CNNLayerCreator() {
     addSpecificCreator({"Parameter"},
```

Comment:
I think this addSpecificCreator invoked multiple times in constructor of CNNLayerCreator.
As far as I remember this is invoked few times during LoadNetwork.
addSpecificCreator add data to the map which is filed of CNNLayerCreator. 
Just wonder if this could be improved by using the same creator instance for all conversion or using this field( creators) as static member.

Reply:
>As far as I remember this is invoked few times during LoadNetwork.
thanks to the pull request, it's no longer

>Just wonder if this could be improved by using the same creator instance for all conversion
it is for now

Reply:
for real world model, (having multiple layers inside), we are going from about 3200 calls to CNNLayerCreator::CNNLayerCreator() down to 1.